etl.py

```diff
@@ -18,12 +18,19 @@
 CARTO_TABLE_NAME = config.get('carto', 'table_name')
 CARTO_DELIMITER = config.get('carto', 'delimiter')
 CARTO_COLUMNS = config.get('carto', 'columns')
+FILE_ENCODING = config.get('etl', 'file_encoding')
 CHUNK_SIZE = int(config.get('etl', 'chunk_size'))
 MAX_ATTEMPTS = int(config.get('etl', 'max_attempts'))
+FORCE_NO_GEOMETRY = config.getboolean('etl', 'force_no_geometry')
 
 api_auth = APIKeyAuthClient(CARTO_BASE_URL, CARTO_API_KEY)
 sql = SQLClient(api_auth)
 bsql = BatchSQLClient(api_auth)
+UTF8 = "utf-8"
+DEFAULT_COORD = 0
```

Comment:
aren't you using this value for incorrect geometries too? wouldn't that conflict with a legit 0 for lat or lon?

Reply:
`DEFAULT_COORD` should be `None`, IMHO

Reply:
+1

Reply:
👍